### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by Kazink36**
Fixes to address #75 and #76

### CODE REVIEW

File: R/cfbd_pbp_data.R [hunk 1]
```diff
@@ -2038,7 +2038,7 @@ prep_epa_df_after <- function(dat) {
 clean_drive_info <- function(drive_df) {
   clean_drive <- drive_df %>%
     dplyr::mutate(
-      pts_drive = dplyr::case_when(
+      drive_pts1 = dplyr::case_when(
```
Comment (akeaswaran) [inline R/cfbd_pbp_data.R:2041]: would like to see these names be more descriptive. eg: drive_pts_rules vs drive_pts_calculated to reflect both ways of calculating them

File: R/cfbd_games.R [hunk 2]
```diff
@@ -1363,7 +1363,8 @@ cfbd_game_team_stats <- function(year,
       }
       df <- df %>%
         tidyr::unnest(.data$teams) %>%
-        tidyr::unnest(.data$stats)
+        tidyr::unnest(.data$stats) %>%
+        dplyr::distinct()
```
Comment (akeaswaran) [inline R/cfbd_games.R:1367]: can you explain how this fixes the long-data issue for a R-layman

Comment (Kazink36) [reply R/cfbd_games.R:1369]: Added a comment but the cause was an error in the pivoting to wide in the next line caused by rare duplicate values from CFBD. Distinct removes the duplicates allowing the rest of the code to run normally. BIll also fixed the duplicates from week 1 in CFBD already so it's not currently an issue, but this should present it popping up in the future

File: R/cfbd_pbp_data.R (hunk 1 above)
Comment (Kazink36) [reply R/cfbd_pbp_data.R]: Good suggestion and added. Also added comments to explain the logic for choosing one over the other that came from looking row-by-row at when they disagreed and why that happened. Most disagreements were just bad drive level data

**Review by akeaswaran — APPROVED** [review]

**Review by saiemgilani — APPROVED** [review]
I have piggy-backed on your PR to fix the most_recent_cfb_season() issue that popped up, as well as outstanding checks issues via tests and documentation updates

**Comment by saiemgilani** [comment]
closes #75 #76

**Pull request merged**